assopy/migrations/0007_rename_assopy_user_model.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+from django.conf import settings
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
+        ('assopy', '0006_add_bank_to_payment_options'),
+    ]
+
+    operations = [
+        migrations.RenameModel(
```

Comment:
does it also rename the db table?

Reply:
Had to double check, but I can confirm it does rename the DB table:

![image](https://user-images.githubusercontent.com/3726919/46570476-1e2dd480-c965-11e8-9907-516468043e5a.png)